README.md

```diff
@@ -106,6 +106,37 @@ you can require the `.node` file with Node and run your tests!
 __Note:__ To create a _Debug_ build of the bindings file, pass the `--debug` (or
 `-d`) switch when running either the `configure`, `build` or `rebuild` command.
 
+#### Usage without Python
+
+There is a [`gyp.js`](https://github.com/indutny/gyp.js/) project, a GYP
+implementation in JavaScript. It generates [`Ninja`](https://ninja-build.org/)
+build files and requires no Python installation.
+
+In this case you will need to install [`Ninja`](https://ninja-build.org/) build
```

Comment:
What about using ninja.js directly instead?

El 2/8/2016 11:22, "Pavel Medvedev" <notifications@github.com> escribió:

> In README.md
> <https://github.com/nodejs/node-gyp/pull/962#discussion_r73122036>:
>
> > @@ -106,6 +106,37 @@ you can require the `.node` file with Node and run your tests!
> >  __Note:__ To create a _Debug_ build of the bindings file, pass the `--debug` (or
> >  `-d`) switch when running either the `configure`, `build` or `rebuild` command.
> >
> > +#### Usage without Python
> > +
> > +There is a [`gyp.js`](https://github.com/indutny/gyp.js/) project, a GYP
> > +implementation in JavaScript. It generates [`Ninja`](https://ninja-build.org/)
> > +build files and requires no Python installation.
> > +
> > +In this case you will need to install [`Ninja`](https://ninja-build.org/) build
>
> There should be a fallback to ninja.js in doWhich() checks, when ninja
> was not found.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node-gyp/pull/962/files/1e8a221ef69573949fc288102f1ba4bb7d048a7e#r73122036>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgfvjh510_d3fOgLcPJXTY51EP1ACYgks5qbww6gaJpZM4I5EHk>
> .
>